package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "pod1612",
```

Should we rename this before merging this PR?

```suggestion
  "name": "beecast",
```